Index: first-mod-after-moving-button-component
===================================================================
--- first-mod-after-moving-button-component
+++ first-mod-after-moving-button-component
@@ -1,48 +1,42 @@
 apiVersion: v3
 definitions:
   extensionPoint:
     definition:
-      attachMode: once
-      containerSelector: ul:has(> * > a:contains('navigation'))
+      contexts:
+        - all
+      documentUrlPatterns:
+        - '*://*/*'
       isAvailable:
         allFrames: true
         matchPatterns:
-          - https://pbx.vercel.app/*
+          - '*://*/*'
         selectors: []
         urlPatterns: []
-      position: append
       reader:
         - '@pixiebrix/document-metadata'
         - '@pixiebrix/document-context'
         - element: '@pixiebrix/html/element'
-      targetMode: document
-      template: <li><a href="#">{{{ caption }}}</a></li>
-      type: menuItem
+      targetMode: eventTarget
+      type: contextMenu
     kind: extensionPoint
 extensionPoints:
   - config:
-      action:
-        - config: {}
-          id: '@pixiebrix/confetti'
-          root: null
-          rootMode: document
-      caption: Action
-      dynamicCaption: false
+      action: []
       onSuccess: true
-      synchronous: false
+      title: Context menu item
     id: extensionPoint
-    label: Test Button Component
+    label: New Context Menu Item
     services: {}
 kind: recipe
 metadata:
   description: First test mod with multiple components
   id: '@extension-e2e-test-unaffiliated/test-mod-created-with-button'
   name: Test Mod created with Button
   version: 1.0.0
 options:
   schema:
     properties: {}
     type: object
   uiSchema:
     ui:order:
       - '*'
