Index: add-quick-bar-starter-brick-to-mod
===================================================================
--- add-quick-bar-starter-brick-to-mod
+++ add-quick-bar-starter-brick-to-mod
@@ -19,83 +19,109 @@
       type: trigger
     kind: extensionPoint
   extensionPoint2:
     definition:
       attachMode: once
       containerSelector: ul:has(> * > a:contains('navigation'))
       isAvailable:
         allFrames: true
         matchPatterns:
           - https://pbx.vercel.app/*
         selectors: []
         urlPatterns: []
       position: append
       reader:
         - '@pixiebrix/document-metadata'
         - '@pixiebrix/document-context'
         - element: '@pixiebrix/html/element'
       targetMode: document
       template: <li><a href="#">{{{ caption }}}</a></li>
       type: menuItem
     kind: extensionPoint
   extensionPoint3:
     definition:
       contexts:
         - all
       documentUrlPatterns:
         - '*://*/*'
       isAvailable:
         allFrames: true
         matchPatterns:
           - '*://*/*'
         selectors: []
         urlPatterns: []
       reader:
         - '@pixiebrix/document-metadata'
         - '@pixiebrix/document-context'
         - element: '@pixiebrix/html/element'
       targetMode: eventTarget
       type: contextMenu
     kind: extensionPoint
+  extensionPoint4:
+    definition:
+      contexts:
+        - all
+      documentUrlPatterns:
+        - '*://*/*'
+      isAvailable:
+        allFrames: true
+        matchPatterns:
+          - '*://*/*'
+        selectors: []
+        urlPatterns: []
+      reader:
+        - '@pixiebrix/document-metadata'
+        - '@pixiebrix/document-context'
+        - element: '@pixiebrix/html/element'
+        - '@pixiebrix/selection'
+      targetMode: eventTarget
+      type: quickBar
+    kind: extensionPoint
 extensionPoints:
   - config:
       action: []
     id: extensionPoint
     label: Test Trigger 00000000-0000-0000-0000-000000000000
     services: {}
   - config:
       action: []
       caption: Action
       dynamicCaption: false
       onSuccess: true
       synchronous: false
     id: extensionPoint2
     label: My pbx.vercel.app button
     services: {}
   - config:
       action: []
       onSuccess: true
       title: Context menu item
     id: extensionPoint3
     label: Context menu item
+    services: {}
+  - config:
+      action: []
+      title: Quick Bar item
+    id: extensionPoint4
+    label: Quick Bar item
     permissions:
       origins: []
       permissions: []
     services: {}
 kind: recipe
 metadata:
   description: Created with the PixieBrix Page Editor
   id: >-
     @extension-e2e-test-unaffiliated/test-trigger-00000000-0000-0000-0000-000000000000-00000000-0000-0000-0000-000000000000
   name: Test Trigger 00000000-0000-0000-0000-000000000000
   version: 1.0.0
 options:
   schema:
     properties: {}
     type: object
   uiSchema:
     ui:order:
       - '*'
 variables:
   schema:
     properties: {}
     type: object
