Index: add-sidebar-panel-starter-brick-to-mod
===================================================================
--- add-sidebar-panel-starter-brick-to-mod
+++ add-sidebar-panel-starter-brick-to-mod
@@ -39,85 +39,137 @@
     kind: extensionPoint
   extensionPoint3:
     definition:
       contexts:
         - all
       documentUrlPatterns:
         - '*://*/*'
       isAvailable:
         allFrames: true
         matchPatterns:
           - '*://*/*'
         selectors: []
         urlPatterns: []
       reader:
         - '@pixiebrix/document-metadata'
         - '@pixiebrix/document-context'
         - element: '@pixiebrix/html/element'
       targetMode: eventTarget
       type: contextMenu
     kind: extensionPoint
   extensionPoint4:
     definition:
       contexts:
         - all
       documentUrlPatterns:
         - '*://*/*'
       isAvailable:
         allFrames: true
         matchPatterns:
           - '*://*/*'
         selectors: []
         urlPatterns: []
       reader:
         - '@pixiebrix/document-metadata'
         - '@pixiebrix/document-context'
         - element: '@pixiebrix/html/element'
         - '@pixiebrix/selection'
       targetMode: eventTarget
       type: quickBar
     kind: extensionPoint
+  extensionPoint5:
+    definition:
+      customEvent: null
+      debounce:
+        leading: false
+        trailing: true
+        waitMillis: 250
+      isAvailable:
+        allFrames: true
+        matchPatterns:
+          - '*://*/*'
+        selectors: []
+        urlPatterns: []
+      reader:
+        - '@pixiebrix/document-metadata'
+        - '@pixiebrix/document-context'
+      trigger: load
+      type: actionPanel
+    kind: extensionPoint
 extensionPoints:
   - config:
       action: []
     id: extensionPoint
     label: Test Trigger 00000000-0000-0000-0000-000000000000
     services: {}
   - config:
       action: []
       caption: Action
       dynamicCaption: false
       onSuccess: true
       synchronous: false
     id: extensionPoint2
     label: My pbx.vercel.app button
     services: {}
   - config:
       action: []
       onSuccess: true
       title: Context menu item
     id: extensionPoint3
     label: Context menu item
     services: {}
   - config:
       action: []
       title: Quick Bar item
     id: extensionPoint4
     label: Quick Bar item
+    services: {}
+  - config:
+      body:
+        - config:
+            body:
+              - children:
+                  - children:
+                      - children:
+                          - config:
+                              heading: h1
+                              title: Example document
+                            type: header
+                        config: {}
+                        type: column
+                    config: {}
+                    type: row
+                  - children:
+                      - children:
+                          - config:
+                              enableMarkdown: true
+                              text: Example text element. **Markdown** is supported.
+                            type: text
+                        config: {}
+                        type: column
+                    config: {}
+                    type: row
+                config: {}
+                type: container
+          id: '@pixiebrix/document'
+          rootMode: document
+      heading: Sidebar Panel
+    id: extensionPoint5
+    label: Sidebar Panel
     permissions:
       origins: []
       permissions: []
     services: {}
 kind: recipe
 metadata:
   description: Created with the PixieBrix Page Editor
   id: >-
     @extension-e2e-test-unaffiliated/test-trigger-00000000-0000-0000-0000-000000000000-00000000-0000-0000-0000-000000000000
   name: Test Trigger 00000000-0000-0000-0000-000000000000
   version: 1.0.0
 options:
   schema:
     properties: {}
     type: object
   uiSchema:
     ui:order:
       - '*'
